robots.txt

```diff
@@ -1,8 +1,4 @@
 User-agent: *
-Disallow: /addons/
-Disallow: /plugins/
 Disallow: /app/
-Disallow: /media/
```

Comment:
I think the only folder that should be removed is the media folder.

The only files I see in there that you might want to disallow are `mautic-form-src.js` which _might_ as well be moved elsewhere and the `dashboard` folder which contains the custom dashboard setups if I'm not wrong.